homeassistant/components/tibber/sensor.py

```diff
@@ -369,7 +369,7 @@ def __init__(self, tibber_home: tibber.TibberHome) -> None:
         """Initialize the sensor."""
         super().__init__(tibber_home=tibber_home)
         self._last_updated: datetime.datetime | None = None
-        self._spread_load_constant = randrange(5000)
+        self._spread_load_constant = randrange(20 * 60)
```

Comment:
Let's assign this to a variable as this is currently a magic number where the meaning of it is anyones guess